components/code-editor/index.js

```diff
@@ -11,6 +11,20 @@ import CodeEditor from './editor';
 import Placeholder from '../../packages/components/src/placeholder';
 import Spinner from '../../packages/components/src/spinner';
 
+/**
+ * Moddule constants
```

Comment:
Typo: "Moddule" -> "Module"

Reply:
This will be wrongly associated with the variable declaration as its JSDoc.,

Reply:
"Module constants" demarcations are useless and encourage a bad practice (a pattern to avoid meaningful documentation of top-level variables).